src/probnum/linalg/solvers/information_ops/_projected_rhs.py

```diff
@@ -0,0 +1,25 @@
+"""Information operator returning a projection of the residual."""
+import numpy as np
+
+import probnum  # pylint: disable="unused-import"
+
+from ._linear_solver_information_op import LinearSolverInformationOp
+
+
+class ProjectedRHSInformationOp(LinearSolverInformationOp):
+    r"""Projected right hand side :math:`s_i \mapsto b^\top s_i = (Ax)^\top s_i` of the linear system.
```

Comment:
I personally think that `s_i^T b` is much easier to understand, because you don't need to transpose the original problem, but rather project it onto `s_i`, similar to a test function